npm test envify error

### Summary:
Running ```npm test``` gives an error:

```
ERROR [karma]: Error: Cannot find module 'envify' from '/Users/dsiwiec/workspace/private/js/react/react-modal'
```
### Steps to reproduce:

 1. Clone the react-modal repo
 2. npm install
 3. npm test

### Expected behavior:
Tests should run and succeed.

### Additional notes:
```
npm -v
3.8.9
```

```
node -v
v6.2.0
```

--- a/package.json
+++ b/package.json
@@ -25,6 +25,7 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.5.0",
+    "envify": "^3.4.1",
     "expect": "1.10.0",
     "karma": "^0.13.22",
     "karma-browserify": "^4.2.1",

